src/config/types.ts

```diff
@@ -0,0 +1,102 @@
+export type XMbSpec = {
+  "account-id": string;
+  assistant: {
+    name: string;
+    description: string;
+    instructions: string;
+    tools?: XMbSpecTools[];
+    chainIds?: number[];
+    categories?: string[];
+    version?: string;
+    repo?: string;
+  };
+  email?: string;
+};
+
+type XMbSpecTools = {
+  type: string;
+};
+
+export type VerifyData = {
+  accountId: string;
+  email: string;
+  repo: string;
+  version?: string;
+  categories?: string[];
+  chainIds?: number[];
+};
+
+export function validateXMbSpec(xMbSpec: unknown): asserts xMbSpec is XMbSpec {
```

Comment:
Nit: type guards usually follow the pattern 

Example: https://github.com/BitteProtocol/near-safe/blob/6637da030c3f022029618c4993ec21be1cbaf655/src/types/guards.ts#L6


```suggestion
export function isXMbSpec(xMbSpec: unknown): asserts xMbSpec is XMbSpec {
```

Might also use `data` for input var name.

Reply:
Also, type guards usually return true or false... in this case we return undefined when it passes all the assertions...